frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -150,13 +155,15 @@ const GlobalConfigPage_: React.FC<GlobalConfigPageProps & GlobalConfigPageExtens
       }
     });
     return () => (isSubscribed = false);
-  }, [configResources, globalConfigs, t]);
+  }, [clusterOperatorConfigResources, configResources, globalConfigs, t]);
   const visibleItems = items.filter(({ label, description = '' }) => {
     return (
       fuzzyCaseInsensitive(textFilter, label) ||
       description.toLowerCase().indexOf(textFilter.toLowerCase()) !== -1
     );
   });
+  const groupedItems = _.groupBy(visibleItems, _.property('label'));
+  const isDup = (item) => _.get(groupedItems, item)?.length > 1;
```

Comment:
nit: Prefer optional chaining

```suggestion
  const isDup = (item) => groupedItems?.item?.length > 1;
```

Reply:
Done.

Reply:
Oops I meant `groupedItems?.[item]?.length > 1`, but looks like you caught my bug